src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/comments/DumpCommentInFlowStyleTest.kt

```diff
@@ -31,7 +28,7 @@ class DumpCommentInFlowStyleTest : FunSpec({
 
     test("flow with comments") {
         val loader = Compose(LoadSettings.builder().setParseComments(true).build())
-        val content = "{ url: text # comment breaks it\n}"
+        val content = "{url: text # comment breaks it\n}"
```

Comment:
@krzema12 do you know why the space was removed? Looks like a redundant change (even if it was done in the original commit)

Reply:
Yeah, looks suspicious. I'm afraid I won't be able to chase it down myself, though...

Reply:
Should we create a ticket for this in such case? I will not be able to have a look at it myself any time soon either

Reply:
Sure: https://github.com/krzema12/snakeyaml-engine-kmp/issues/512